app/riot/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@storybook/riot",
+  "version": "4.0.0-alpha.18",
+  "description": "Storybook for riot.js: View riot snippets in isolation with Hot Reloading.",
+  "homepage": "https://github.com/storybooks/storybook/tree/master/app/riot",
+  "bugs": {
+    "url": "https://github.com/storybooks/storybook/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybooks/storybook.git"
+  },
+  "license": "MIT",
+  "main": "dist/client/index.js",
+  "bin": {
+    "build-storybook": "./bin/build.js",
+    "start-storybook": "./bin/index.js",
+    "storybook-server": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "node ../../scripts/prepare.js"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0-rc.2",
+    "@storybook/core": "4.0.0-alpha.18",
+    "common-tags": "^1.8.0",
+    "global": "^4.3.2",
+    "moment": "^2.22.2",
+    "raw-loader": "^0.5.1",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2"
+  },
+  "devDependencies": {
+    "cross-env": "^5.2.0",
+    "@babel/preset-flow": "^7.0.0-rc.2",
+    "@babel/preset-react": "^7.0.0-rc.2",
+    "@babel/runtime": "^7.0.0-rc.2",
+    "@babel/preset-env": "^7.0.0-rc.2",
+    "@babel/plugin-transform-runtime": "^7.0.0-rc.2",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0-rc.2"
+  },
+  "peerDependencies": {
+    "babel-core": "^7.0.0 || ^8.0.0 || ^8.0.0-beta.6",
```

Comment:
It should be `"babel-loader": "^7.0.0 || ^8.0.0 || ^8.0.0-beta.6",`

`babel-core` is a peer of babel-loader < 8
`@babel/core` is a peer of babel-loader >= 8

So to allow people use an older babel version, we recently moved to a peer of `babel-loader` 🤷‍♂️

Reply:
changed